Simplenote/src/main/java/com/automattic/simplenote/NotesActivity.java

```diff
@@ -643,6 +644,20 @@ public boolean onOptionsItemSelected(MenuItem item) {
             return true;
         }
         switch (item.getItemId()) {
+            case R.id.menu_sidebar:
+                FragmentManager fm = getSupportFragmentManager();
+                FragmentTransaction ft = fm.beginTransaction();
+                if (mNoteListFragment.isHidden()) {
+                    ft.setTransition(FragmentTransaction.TRANSIT_FRAGMENT_OPEN);
+                    ft.show(mNoteListFragment);
+                } else {
+                    ft.setTransition(FragmentTransaction.TRANSIT_FRAGMENT_FADE);
+                    ft.hide(mNoteListFragment);
+                }
+                ft.commit();
+                fm.executePendingTransactions();
```

Comment:
I think that makes sense. Also, I changed to use `ft.commitNowAllowingStateLoss()` to align with other similar use cases of hiding/showing the note list.

According to docs `commitNowAllowingStateLoss()` is:
> Like `commitNow()` but allows the commit to be executed after an activity's state is saved.  This is dangerous because the commit can be lost if the activity needs to later be restored from its state, so this should only be used for cases where it is okay for the UI state to change unexpectedly on the user.

Which I think is fine in this particular use case.